Source/EventFlow.ReadStores.ElasticSearch/EsConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace EventFlow.ReadStores.ElasticSearch
+{
+    public class EsConfiguration : IEsConfiguration
```

Comment:
@tabro There's no need for the `IEsConfiguration` as it isn't used anywhere but in the `ConfigureElasticSearch`